### PR TITLE
Align mk22_lab_advanced model with actual stacklight state

### PR DESCRIPTION
**Pull request by dkalashnik**

### CODE REVIEW

File: classes/cluster/mk22_lab_advanced/stacklight/init.yml [hunk 1]
```diff
@@ -5,14 +5,16 @@ parameters:
...
+    grafana_password: password
     grafana_influxdb_host: ${_param:stacklight_monitor_address}
     elasticsearch_port: 9200
     influxdb_stacklight_password: lmapass
+    influxdb_admin_password: password
     influxdb_port: 8086
     influxdb_database: lma
     influxdb_user: lma
     influxdb_password: lmapass
+    kibana_port: 5601
```
Comment (thouveng) [inline classes/cluster/mk22_lab_advanced/stacklight/init.yml]: IIUC ports are hardcoded  now. See #152 
So I think that we can just remove it.

File: classes/cluster/mk22_lab_advanced/stacklight/init.yml [hunk 2]
```diff
@@ -5,14 +5,16 @@ parameters:
     heka_aggregator_host: ${_param:stacklight_monitor_address}
     aggregator_port: 5565
     grafana_user: admin
-    mysql_grafana_password: password
```
Comment (SwannCroiset) [inline classes/cluster/mk22_lab_advanced/stacklight/init.yml]: I think you need both, on for mysql user another for grafana user

Comment (thouveng) [inline classes/cluster/mk22_lab_advanced/stacklight/init.yml]: It has been moved in openstack/init.yml

File: classes/cluster/mk22_lab_advanced/stacklight/init.yml (hunk 1 above)
Comment (dkalashnik) [inline classes/cluster/mk22_lab_advanced/stacklight/init.yml]: @thouveng Done, also I've removed elastic port

File: classes/cluster/mk22_lab_advanced/openstack/init.yml [hunk 3]
```diff
@@ -6,9 +6,6 @@ parameters:
     cluster_public_host: _
     opencontrail_version: 3.0
     opencontrail_compute_dns: 8.8.8.8
-    opencontrail_compute_gateway: 172.16.10.1
-    opencontrail_compute_address: ${_param:single_address}
-    opencontrail_compute_iface: eth1
```
Comment (dkalashnik) [inline classes/cluster/mk22_lab_advanced/openstack/init.yml]: Moved to openstack/compute.yml

**Pull request merged**